src/test/basic.test.js

```diff
@@ -176,5 +176,18 @@ describe('basic', () => {
 
       expectCSSMatches('.sc-a {} .d { color:red; } .sc-b {} .c { color:blue; }')
     })
+
+    it('handle media at-rules inside style rules', () => {
+      const Comp = styled.div`
+        > * {
+          @media (min-width: 500px) {
+            color: pink;
+          }
+        }
+      `
+
+      shallow(<Comp />)
+      expectCSSMatches('.sc-a{ } @media (min-width:500px){ .b > *{ color:pink; } } ')
```

Comment:
Is this right? Shouldn’t it be .sc-a > * inside the media query?

Reply:
It's indeed correct 😬 

Unfortunately at-rule are supposed to wrap normal style rules, but often browsers are quite lenient with these kinds of restrictions, since CSS is supposed to be interpreted in a non-restrictive way.

But `insertRule` will be more strict with these kinds of mistakes, so we have to reverse the order. This was already in place by stylis-rule-sheet, but it'd insert an extra closing curly brace before